.gitmodules

```diff
@@ -22,6 +22,7 @@
 [submodule "curl/curlpp"]
 	path = curl/curlpp
 	url = https://github.com/jpbarrette/curlpp.git
+	branch = master
```

Comment:
Is it necessary to specify branch here? Leaving branch empty will make the project immune to upstream changes unless manually updated

Reply:
I've had some issues getting newer package versions from that project, but specifying it in the file is no longer necessary.